auth/auth.go

```diff
@@ -322,27 +322,28 @@ func (c *baseClient) verifyIDToken(ctx context.Context, idToken string, checkRev
 	}
 
 	if c.isEmulator || checkRevokedOrDisabled {
-		revoked, disabled, err := c.checkRevokedOrDisabled(ctx, decoded)
+		errMessageDisabled := &internal.FirebaseError{
+			ErrorCode: internal.InvalidArgument,
+			String:    "User has been disabled",
```

Comment:
Go convention is to start off error messages with lowercase letters (unless acronym or other shorthand like "ID").